site/content/docs/5.3/components/list-group.md

```diff
@@ -202,23 +218,19 @@ Contextual classes also work with `.list-group-item-action` for `<a>` and `<butt
 
 Add badges to any list group item to show unread counts, activity, and more with the help of some [utilities]({{< docsref "/utilities/flex" >}}).
 
-{{< ods-incompatibility-alert >}}
-This variant should not be used because it does not respect the Orange Design System specifications.
-{{< /ods-incompatibility-alert >}}
-
 {{< example >}}
 <ul class="list-group">
   <li class="list-group-item d-flex justify-content-between align-items-center">
     A list item
-    <span class="badge bg-primary rounded-pill">14</span>
+    <span class="badge bg-info text-white rounded-pill">14</span>
```

Comment:
It seems that we have some problems with the blue badges: not the right shape, not the right spacing and not the right blue either.

This point will be for the design review.

Reply:
Haven't changed the spacing for those since the issue comes from the badges themselves since, here the issue was the to change the bg-color and color. I think it might be tackled outside of this PR anyway.